internals/secrethub/account_inspect.go

```diff
@@ -40,38 +43,84 @@ func (cmd *AccountInspectCommand) Run() error {
 		return err
 	}
 
-	user, err := client.Users().Me()
+	account, err := client.Accounts().Me()
 	if err != nil {
 		return err
 	}
-
-	output, err := cli.PrettyJSON(newOutputUser(user, cmd.timeFormatter))
-	if err != nil {
-		return err
+	var output string
+	if account.AccountType == accountTypeUser {
+		user, err := client.Users().Me()
+		if err != nil {
+			return err
+		}
+		output, err = cli.PrettyJSON(newOutputUser(user, cmd.timeFormatter))
+		if err != nil {
+			return err
+		}
+	} else if account.AccountType == accountTypeService {
+		service, err := client.Services().Get(account.Name.String())
+		if err != nil {
+			return err
+		}
+		output, err = cli.PrettyJSON(newOutputService(service, account, cmd.timeFormatter))
+		if err != nil {
+			return err
+		}
 	}
-
 	fmt.Fprintln(cmd.io.Output(), output)
 
 	return nil
 }
 
+// outputAccount contains the fields common in both outputUser and outputService
+type outputAccount struct {
+	AccountType      string
+	AccountName      string
+	PublicAccountKey []byte `json:",omitempty"`
+}
+
 // outputUser is a user friendly JSON representation of a user account.
 type outputUser struct {
-	Username         string
-	FullName         string
-	Email            string `json:",omitempty"`
-	EmailVerified    bool   `json:",omitempty"`
-	CreatedAt        string `json:",omitempty"`
-	PublicAccountKey []byte `json:",omitempty"`
+	Username      string
+	FullName      string
+	Email         string `json:",omitempty"`
+	EmailVerified bool   `json:",omitempty"`
+	CreatedAt     string `json:",omitempty"`
+	outputAccount
 }
 
 func newOutputUser(user *api.User, timeFormatter TimeFormatter) *outputUser {
 	return &outputUser{
-		Username:         user.Username,
-		FullName:         user.FullName,
-		Email:            user.Email,
-		EmailVerified:    user.EmailVerified,
-		CreatedAt:        timeFormatter.Format(user.CreatedAt.Local()),
-		PublicAccountKey: user.PublicKey,
+		Username:      user.Username,
+		FullName:      user.FullName,
+		Email:         user.Email,
+		EmailVerified: user.EmailVerified,
+		CreatedAt:     timeFormatter.Format(user.CreatedAt.Local()),
+		outputAccount: outputAccount{
+			AccountType:      accountTypeUser,
+			AccountName:      user.Username,
+			PublicAccountKey: user.PublicKey,
+		},
+	}
+}
+
+// outputService is a user friendly JSON representation of a service account.
+type outputService struct {
+	Description string
+	CreatedBy   string
+	CreatedAt   string `json:",omitempty"`
+	outputAccount
+}
+
+func newOutputService(service *api.Service, account *api.Account, timeFormatter TimeFormatter) *outputService {
+	return &outputService{
+		Description: service.Description,
+		CreatedBy:   service.CreatedBy.String(),
```

Comment:
This will be a `uuid` is this an issue?

Reply:
I don't think an UUID has any use for a user. So what can we do to solve that?

Reply:
Thought so too. Having the username there would be way more useful, but to do that we'd have to either:
1. Modify the server controller for this request to (also) send the username.
2. Add another route to the server that gets a user by its id and call that (which sounds like a pretty bad idea to me)

What do you think?

Reply:
Another option I just thought of: we can also leave it out altogether (at least for now). I think the use-case for getting this info about the service account you're currently using, is pretty limited.

Reply:
Good point. I agree, let's leave it out for now.